### PR TITLE
Fix yettel_bg spider

### LINKED ISSUE #6323: yettel_bg is broken
The spider does not return anything. It seems like they've moved the page for getting their store data. I wasn't able to find the new address or how the data reaches their store locator page.

Store locator: https://www.yettel.bg/faq/digital-customer-service/store-locator

Spider: https://github.com/alltheplaces/alltheplaces/blob/master/locations/spiders/yettel_bg.py

### PR DESCRIPTION
**Pull request by Dimitar5555**
Resolves #6323

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -39,3 +39,4 @@ unidecode = "*"
 pygeohash = "*"
 scrapy-zyte-api = "*"
 cryptography = "*"
+openpyxl = "*"
```
Comment (davidhicks) [inline Pipfile:42]: You'll need to run `pip lock` to update `Pipfile.lock` and then add to this PR.

**Review by davidhicks — CHANGES_REQUESTED** [review]

File: locations/spiders/yettel_bg.py [hunk 2]
```diff
@@ -10,20 +14,45 @@ class YettelBGSpider(Spider):
         "brand_wikidata": "Q14915070",
         "country": "BG",
     }
-    start_urls = ["https://www.yettel.bg/store-locator/json"]
+    start_urls = ["https://www.yettel.bg/dA/1d9e589cca"]
```
Comment (Cj-Malone) [inline locations/spiders/yettel_bg.py:17]: The spreadsheet has already changed to ``.

I recommend starting with the store finder and extracting the url from there:
```suggestion
    start_urls = ["https://www.yettel.bg/faq/digital-customer-service/store-locator"]
```

**Comment by CloCkWeRX** [comment]
Conflicts at the moment.

If you need a quick instance, maybe start a code space for this branch. Will be clunky but let you check everything is happy

**Comment by Dimitar5555** [comment]
It seems to work as expected now.

**Comment by iandees** [comment]
It looks like you might have been running something other than Python 3.11 when you updated the Pipfile. Can you double-check that?

**Comment by Dimitar5555** [comment]
> It looks like you might have been running something other than Python 3.11 when you updated the Pipfile. Can you double-check that?

It was 3.10. Should be fixed now.

**Pull request merged**